.gitignore

```diff
@@ -19,3 +19,6 @@ test-data/*
 .vscode
 *.sublime-project
 *.sublime-workspace
+
+# DevOops Magic
+/.token
```

Comment:
While this is funny when you look at it as a diff, it becomes a strange code (or config) to read later. The comment isn't really informative. I think you are excluding `.token` from git because `wget` doesn't direct to stdout by default like `curl` does, but saves to a file. Since you switched to `curl`, it not necessary to ignore `.token`.